pkgs/by-name/um/umu-launcher/package.nix

```diff
@@ -2,13 +2,15 @@
   buildFHSEnv,
   lib,
   umu-launcher-unwrapped,
+  extraPkgs ? pkgs: [ ],
+  extraLibraries ? pkgs: [ ],
 }:
 buildFHSEnv {
   pname = "umu-launcher";
   inherit (umu-launcher-unwrapped) version meta;
 
-  # Use umu-launcher-unwrapped from the package args, to simplify overriding
-  targetPkgs = _: [ umu-launcher-unwrapped ];
+  targetPkgs = pkgs: [ pkgs.umu-launcher-unwrapped ] ++ extraPkgs pkgs;
+  multiPkgs = extraLibraries;
```

Comment:
Needs to take https://github.com/NixOS/nixpkgs/pull/375588 into account:
```suggestion
  # Use umu-launcher-unwrapped from the package args, to simplify overriding
  targetPkgs = pkgs: [ umu-launcher-unwrapped ] ++ extraPkgs pkgs;
  multiPkgs = extraLibraries;
```

Maybe this formatting would be nicer, though:
```nix
  # Use umu-launcher-unwrapped from the package args, to simplify overriding
  targetPkgs = pkgs: extraPkgs pkgs ++ [
    umu-launcher-unwrapped
  ];
  multiPkgs = extraLibraries;
```

Reply:
Unsure why this comment is necessary? I'm truthfully unsure how often you would need to override the unwrapped umu unless you have your own patches or something.

Reply:
The comment is really just to clarify that _not_ using `pkgs` is deliberate.

No comment is ever strictly necessary, but they're often helpful :slightly_smiling_face:

Reply:
The comment wasn't removed on purpose, just got removed while I was fixing the merge conflict on accident. Just wasn't sure why it was there.

Reply:
I've opted to simply place the comment back where it was, instead of the suggested formatting.